modules/layers/src/scatterplot-layer/scatterplot-layer-fragment.glsl.js

```diff
@@ -25,6 +25,7 @@ precision highp float;
 
 uniform bool filled;
 uniform float stroked;
+uniform bool smoothEdges;
```

Comment:
There are two usages of `smoothedge` in this shader, both need to be replaced with `step` if the toggle is off (FirstPersonView use case).